source/ada/lsp-ada_handlers.adb

```diff
@@ -531,6 +531,30 @@ package body LSP.Ada_Handlers is
       LSP.File_Monitors.Unchecked_Free (Self.File_Monitor);
    end Cleanup;
 
+   ----------------
+   -- Clean_Logs --
+   ----------------
+
+   procedure Clean_Logs (Self : access Message_Handler; Dir : Virtual_File) is
+      Log_Files : File_Array_Access := Read_Dir (Dir, Files_Only);
+      Dummy     : Boolean;
+      Cpt       : Integer := 0;
+   begin
+      Sort (Log_Files.all);
+      --  Browse the log files in reverse timestamp order
+      for F of reverse Log_Files.all loop
+         --  Filter out files like traces.cfg
+         if GNATCOLL.Utils.Starts_With (+F.Base_Name, "als.") then
```

Comment:
You should filter with .txt instead: the log filenames may be changed in traces.cfg

Reply:
The default name is finishing by ".log" and we are overriding the name in traces.cfg when using ALS_HOME

Reply:
Then check for '.log' no ? Also, will it conflict with the GS preference?